app/views/layouts/application.html.erb

```diff
@@ -7,8 +7,8 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application' %>
```

Comment:
Removed the Turbolinks-related arguments here, and all the articles I read about making sure your styles work on Heroku said to use a `stylesheet_pack_tag` instead of a `stylesheet_link_tag`.